index.html

```diff
@@ -35,6 +35,7 @@
                 'example/imagery',
                 'example/eventGenerator',
                 'example/styleguide'
+
```

Comment:
Small point, but it can be helpful to avoid unintentional whitespace-only changes like this; they add some noise to the diffs and the file history.